tests/baselines/reference/javascriptThisAssignmentInStaticBlock.js

```diff
@@ -30,11 +30,11 @@ _a = Thing;
 })();
 Thing.doSomething();
 // GH#46468
-class ElementsArray extends Array {
+class ElementsArray extends (_c = Array) {
 }
 _b = ElementsArray;
 (() => {
-    const superisArray = super.isArray;
+    const superisArray = Reflect.get(_c, "isArray", _b);
     const customIsArray = (arg) => superisArray(arg);
     _b.isArray = customIsArray;

```

Comment:
Another interesting diff to check.

Reply:
Only this way because I incorrectly removed the es2015 transform that previously handled this later. Oops.